examples/transloadit/main.js

```diff
@@ -17,23 +28,35 @@ const TRANSLOADIT_KEY = '35c1aed03f5011e982b6afe82599b6a0'
 const TEMPLATE_ID = 'bbc273f69e0c4694a5a9d1b587abc1bc'
 
 /**
- * robodog.form
+ * Form
  */
 
-const formUppy = robodog.form('#test-form', {
+const formUppy = new Uppy({
   debug: true,
-  fields: ['message'],
+  autoProceed: true,
   restrictions: {
     allowedFileTypes: ['.png'],
   },
-  waitForEncoding: true,
-  params: {
-    auth: { key: TRANSLOADIT_KEY },
-    template_id: TEMPLATE_ID,
-  },
-  modal: true,
-  progressBar: '#test-form .progress',
 })
+  .use(Dashboard, {
```

Comment:
The next example is called "Form with Dashboard" so I would expect this is without Dashboard? Or delete this example altogether.